e2e-network/docker/test-05-simple-k8s.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+set -e
+
+TEST_TMP="$(rm -rf "$0.tmpdir" && mkdir -p "$0.tmpdir" && (cd "$0.tmpdir" && pwd))"
+TEST_LOGS="$(mkdir -p "$0.logs" && (cd "$0.logs" && pwd))"
+FABLO_HOME="$TEST_TMP/../.."
+
+networkUp() {
+  "$FABLO_HOME/fablo-build.sh"
+  (cd "$TEST_TMP" && "$FABLO_HOME/fablo.sh" init kubernetes node)
```

Comment:
I believe the whole file should be removed, because it resides in docker directory